### PR TITLE
handle mtime via obsinfo

### PR DESCRIPTION
**Pull request by adrianschroeter**

### CODE REVIEW

**Comment by aspiers** [comment]
LGTM, but could you add some tests?

File: tar_scm.py [hunk 1]
```diff
@@ -1259,7 +1259,13 @@ def singletask(use_obs_scm, args):
 
     repodir = None
     # construct repodir (the parent directory of the checkout)
-    if repocachedir and os.path.isdir(os.path.join(repocachedir, 'repo')):
+    if repocachedir:
+        # construct subdirs on very first run
+        if not os.path.isdir(os.path.join(repocachedir, 'repo')):
+            os.mkdir(os.path.isdir(os.path.join(repocachedir, 'repo')))
```
Comment (aspiers) [inline tar_scm.py]: This looks wrong to me - presumably you don't want the `isdir` inside the `mkdir` call.

File: tar_scm.py [hunk 2]
```diff
@@ -1259,7 +1259,13 @@ def singletask(use_obs_scm, args):
...
+        if not os.path.isdir(os.path.join(repocachedir, 'incoming')):
+            os.mkdir(os.path.isdir(os.path.join(repocachedir, 'incoming')))
```
Comment (aspiers) [inline tar_scm.py]: Same here.

File: tar_scm.py (hunk 1 above)
Comment (adrianschroeter) [inline tar_scm.py]: yep ... fixed

**Comment by M0ses** [comment]
should be obsoleted by https://github.com/openSUSE/obs-service-tar_scm/pull/123

**Comment by adrianschroeter** [comment]
yes, lets close this.

**Pull request closed without merging**